src/_zkapauthorizer/tahoe.py

```diff
@@ -51,7 +50,7 @@ async def decorated(*a, **kw) -> Awaitable:
     return retry_decorator
 
 
-def _not_enough_servers(exc: Exception) -> bool:
+def _not_enough_servers(exc: Exception, /) -> bool:
```

Comment:
This makes `exc` positional-only which is apparently required for this type to match `Callable[[Exception], bool]` - which is what is required by the API this is passed to.

It looks like newer Python might give you a way to describe keyword-argument functions but we'll have to wait for that.